<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ELF layout viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="layoutView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
    <addaction name="actionZoom_original"/>
    <addaction name="actionZoom_best"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="actionZoom_original"/>
   <addaction name="actionZoom_best"/>
  </widget>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset theme="zoom-in"/>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset theme="zoom-out"/>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionZoom_original">
   <property name="icon">
    <iconset theme="zoom-original"/>
   </property>
   <property name="text">
    <string>Zoom 100%</string>
   </property>
  </action>
  <action name="actionZoom_best">
   <property name="icon">
    <iconset theme="zoom-fit-best"/>
   </property>
   <property name="text">
    <string>Zoom best</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
